doc/source/serve/http-guide.md

```diff
@@ -1,13 +1,36 @@
-# Calling Deployments via HTTP
+# HTTP with Serve
 
 
+
+This section should help you understand how to:
+
+- configure your HTTP server location
+- send HTTP requests to Serve deployments
+- use Ray Serve to integrate with FastAPI
+- use customized HTTP Adapters
+
+
+## Configuring HTTP Server Locations
+
+By default, Ray Serve starts a single HTTP server on the head node of the Ray cluster.
+You can configure this behavior using the `http_options={"location": ...}` flag
+in {mod}`serve.run <ray.serve.run>`:
+
+- **HeadOnly**: start one HTTP server on the head node. Serve
+  assumes the head node is the node you executed `serve.run`
+  on. This is the default.
+- **EveryNode**: start one HTTP server per node.
+- **NoServer** or `None`: disable HTTP server.
+
+:::{note}
+To achieve high availability of Serve's HTTP proxies, use **EveryNode** option to point to the instance group of Ray cluster
+:::
+
 (serve-http)=
 
 ## Calling Deployments via HTTP
 
-### Basic Example
-
-When you create a deployment, it is exposed over HTTP by default at `/{deployment_name}`. You can change the route by specifying the `route_prefix` argument to the {mod}`@serve.deployment <ray.serve.api.deployment>` decorator.
+When you create a deployment, Serve exposes your deployment over HTTP by default at `/`. You can change the route by specifying the `route_prefix` argument to the {mod}`@serve.deployment <ray.serve.api.deployment>` decorator.
```

Comment:
```suggestion
When you deploy a Serve application, the ingress deployment (the one passed to `serve.run`) will be exposed over HTTP. If you want to route to another deployment, you can do so using the `ServeHandle` API <link to model comopsition>.
```

using `route_prefix` is no longer recommended! we want everything going through the single ingress